tests/test_core/test_alignment.py

```diff
@@ -2763,6 +2774,9 @@ def test_counts_per_seq(self):
         a = self.a
         f = a.counts_per_seq()
         self.assertEqual(f.array, array([[3, 1], [1, 3]]))
+        f = a.counts_per_seq(exclude_unobserved=True)
+        self.assertEqual(f.array, array([[3, 1], [1, 3]]))
```

Comment:
needs to consider case where there's an unobserved state. Easily achieved by setting `motif_length=2`